crates/cxx-qt-gen/src/generator/naming/functions.rs

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use convert_case::{Case, Casing};
+use quote::format_ident;
+use syn::Ident;
+
+use super::*;
```

Comment:
should this just be `super::CombinedIdent` ?

Reply:
darn it, this again :see_no_evil:

Reply:
Done